wiki/Registration/en.md

```diff
@@ -19,3 +21,15 @@ Don't forget to read the [rules](/wiki/Rules) very carefully!
 Once that is done, you are now ready to start your rhythm adventure! You can [add beatmaps](/wiki/Installation#adding-beatmapsets) to play, [make a skin](/wiki/Skinning), or [make a beatmap](/wiki/Beatmapping). You could also stop by and say "Hi" in the [Introductions subforum](https://osu.ppy.sh/community/forums/8).
 
 If you need further in-game help, you can ask in the `#help` [Internet Relay Chat](/wiki/Internet_Relay_Chat) (IRC) channel by opening the chat console (press `F8` or `F9` in-game) then type `/join help`. You could also post your question in the [Help subforum](https://osu.ppy.sh/community/forums/5).
+
+## Troubleshooting
+
+### I receive some sort of server-side error when trying to play online!
+
+**This can happen if you are using a shared internet connection, a proxy, or VPN.**
```

Comment:
the reasons listed here are not the only reason this can happen. it could be a server-side error, meaning the servers are having issues. ie. what the error actually reads as.

Reply:
added a reference to status.ppy.sh here, that should explain what is going on